pkg/cloudprovider/cloudprovider.go

```diff
@@ -22,11 +22,11 @@ func init() {
 }
 
 func (k *k3s) Initialize(clientBuilder cloudprovider.ControllerClientBuilder, stop <-chan struct{}) {
```

Comment:
One assumes we are keeping this member func around to satisfy an interface implementation. It is really handy to call out such by declaring an unused ephemeral var of the interface type assigned a pointer the struct in question, e.g.:
```
var _ Interface = &k3s{}
```

Reply:
If such isn't the case, we should remove `Initialize()` I think.

Reply:
This is one of the functions required by the provider interface, alongside the others (Instances(), Zones(), etc) down below. Is there an easy way to stub out just this single function?

Reply:
The `var` declaration I suggested would make this more explicit at least.